auth-web/devops/vaults.env

```diff
@@ -10,6 +10,8 @@ VUE_APP_PAYMENT_PAYEE_NAME="op://relationship/$APP_ENV/auth-web/PAYMENT_PAYEE_NA
 VUE_APP_FILE_SERVER_URL="op://relationship/$APP_ENV/auth-web/FILE_SERVER_URL"
 VUE_APP_APPROVE_ACCOUNT_SLA_DAYS="op://relationship/$APP_ENV/auth-web/APPROVE_ACCOUNT_SLA_DAYS"
 VUE_APP_ONE_STOP_URL="op://relationship/$APP_ENV/auth-web/ONE_STOP_URL"
+VUE_APP_SOCIETIES_URL="op://web-url/$APP_ENV/bcregistry/SOCIETIES_URL"
+VUE_APP_CORP_FORMS_URL="op://web-url/$APP_ENV/bcregistry/CORP_FORMS_URL"
```

Comment:
@pwei1018 It looks like these new keys are not being populated in the temp URL (https://bcregistry-account-dev--pr-2422-czkxdizn.web.app/).

Reply:
Never mind - I figured it out.

@chenhongjing , you need to add these 2 new keys to devops/cloudbuild-pr.yaml, same as Karim did here: https://github.com/bcgov/business-create-ui/pull/559/files#r1287700270

Reply:
And then, run the Google Cloud Build again.

Reply:
Did the changes, but I find it can't go to form page. Why does the problem still exist?

Reply:
Patrick needs to configure something for us so that the 1Password keys are correctly converted into env variables and then built into the app. I've asked him to do that just now. This should be fixed by tomorrow morning.